web-console/src/utils/ingestion-spec.tsx

```diff
@@ -2615,6 +2687,10 @@ export function upgradeSpec(spec: any): any {
         deepSet(spec, 'ioConfig.firehose.type', 's3');
         break;
 
+      case 'static-azure':
+        deepSet(spec, 'ioConfig.firehose.type', 'azure');
+        break;
```

Comment:
There is no need for this upgrade spec clause, there was never a `'static-azure'`. This is back compat code but Azure ingestion did not used to exist. You can just remove this change.